taxi/urls.py

```diff
@@ -22,6 +28,12 @@
     path(
         "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
     ),
+    path("cars/create", CarCreateView.as_view(), name="car-create"),
+    path("cars/<int:pk>/update", CarUpdateView.as_view(), name="car-update"),
+    path("cars/<int:pk>/delete", CarDeleteView.as_view(), name="car-delete"),
+    path("manufacturers/create", ManufacturerCreateView.as_view(), name="manufacturer-create"),
+    path("manufacturers/<int:pk>/update", ManufacturerUpdateView.as_view(), name="manufacturer-update"),
```

Comment:
Consider adding a trailing slash to the URL pattern 'manufacturers/<int:pk>/update'. Following Django's convention, a trailing slash should be included.